src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {categories:[], showText: false },
+  reducers: {
+    toggleText(state) {
+      state.showText = !state.showText;
+    },
+  },
+});
+
+export const selectTect = (state) => state.categories;
+export const categoriesActions = categoriesSlice.actions;
+
+export default categoriesSlice.reducer;

```

Comment:
Kindly follow the same structure for setting up the categories reducer as well as per project requirements:

![image](https://user-images.githubusercontent.com/38088818/167386162-fdaac524-57cd-4488-9339-e606c907a4b9.png)

- Remember that the `CHECK_STATUS` action should return `"Under construction"` in the reducer.